.github/ISSUE_TEMPLATE/SIMPLE-TASK.yml

```
@@ -0,0 +1,41 @@
+name: "📋 Rest-Api Task"
```

```suggestion
name: "📋 Rest-API Task"
```
Check your capitalization throughout.